src/Apps/Order/Components/TransactionSummary.tsx

```diff
@@ -31,26 +31,45 @@ export const TransactionSummary: React.SFC<TransactionSummaryProps> = ({
       resized_transactionSummary: { url: imageURL },
     },
   } = lineItems.edges[0].node.artwork
+  const truncateTextStyle = {
+    whiteSpace: "nowrap",
+    overflow: "hidden",
+    textOverflow: "ellipsis",
+    width: "100%",
+  } as any
 
   return (
     <Flex flexDirection="column" {...others}>
       <StackableBorderBox flexDirection="row">
         <Box height="auto">
           <Image src={imageURL} width="55px" mr={1} />
         </Box>
-        <Flex flexDirection="column">
-          <Serif size="2" weight="semibold" color="black60">
+        <Flex
+          flexDirection="column"
+          style={{ width: `calc(100% - 55px - ${space(1)}px)` }} // 55px for Image above, plus 1 for its mr.
```

Comment:
Okay, thanks! I'll take a look at that 👍

Reply:
This fell off my radar for this morning, probably won't get to fixing this before you're done for the day, but I'll have something for you to review tomorrow.

Reply:
Very cool! That worked as you described. 

I've generally found it difficult to find answers to questions like "how do I make a label truncate with an ellipsis?" because the target domain is so vast. Is this a styled-system/styled-components question? React? CSS? HTML? Flexbox? The search space is so huge that I'm having a hard time getting answers to intro-level questions like this while ramping up those skills. I'll post in Slack, maybe other Front-End folks have suggestions.

Reply:
Yeah, I think it's just hard to reframe the domain - FWIW, its CSS you just need to memorize in the end (remember this isn't a `Label` in HTML either, those don't exist in web, just bits of text that don't have a name per-say) 

https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow is what you're looking for